_drafts/2017-05-31-product-goals.md

```diff
@@ -0,0 +1,53 @@
+---
+title:  "Product goals for 2017"
+author: borsi
+tags: [goals, community, roadmap, redesign, statistics]
+---
+## Everybody needs goals
+
+Us, too. In fact we need better goals, than *"making more money"* or *"creating better stuff"*. So last year we decided to be [SMART][bb699076] about it and set high-level product goals for ourselves and for our community. After all, we want to be the best ECM development platform for a large community of developers. But how do we get there?
+
+![What is my purpose?](http://i.imgur.com/EdK1bag.gif)
+
```

Comment:
the image is too big compare to the paragraph and having anim gifts make the reading harder (your eye ball tries to focus on the moving element and difficult to read the text around it).